Author: George Kiagiadakis <kiagiadakis.george@gmail.com>
Description: Fix typo in CMakeLists.txt to link with pthreads.
Origin: Upstream, r1157114.

--- a/kdecore/CMakeLists.txt
+++ b/kdecore/CMakeLists.txt
@@ -346,7 +346,7 @@
 
 if(NOT WIN32)
     set(kdecore_LIB_SRCS ${kdecore_LIB_SRCS} util/kshareddatacache.cpp)
-    set(kdecore_OPTIONAL_LIBS ${kdecore_OPTIONAL_LIBS} ${CMAKE_THREAD_LIB_INIT})
+    set(kdecore_OPTIONAL_LIBS ${kdecore_OPTIONAL_LIBS} ${CMAKE_THREAD_LIBS_INIT})
 else(NOT WIN32)
     set(kdecore_LIB_SRCS ${kdecore_LIB_SRCS} util/kshareddatacache_win.cpp)
     set(kdecore_LIB_SRCS ${kdecore_LIB_SRCS} io/kdirwatch_win.cpp)
